server/src/test/java/org/cloudfoundry/identity/uaa/provider/oauth/ExternalOAuthProviderConfiguratorTests.java

```diff
@@ -107,6 +107,7 @@ void setup() throws MalformedURLException {
         config.setRelyingPartySecret("identitysecret");
         config.setResponseType("id_token");
         config.setScopes(List.of("openid", "cloud_controller.read"));
+        config.setAdditionalAuthzParameters(Map.of("token_format", "jwt"));
```

Comment:
Can you please add a test case to this class that checks whether the additional authorization parameters are added properly to the authentication URLs?